### ISSUE TITLE
Playnite freezes after clicking install in itch.io game

### ISSUE THREAD

**Issue by darklinkpower** [JosefNemec/Playnite#1879]
**Describe the bug**
Playnite crashed after clicking install in itch.io game

**To Reproduce**
Playnite crashed after clicking install in itch.io game

**Diagnostics ID**

```
2020-06-12 22:13:37.5859|INFO:[router] [info] Initiating graceful butlerd shutdown
2020-06-12 22:13:37.5859|INFO:[router] [info] While shutting down, request 402510343 has completed
2020-06-12 22:13:37.5859|INFO:[router] [info] No in-flight requests left, we can shut down now.
2020-06-12 22:13:37.5859|DEBUG:Butler: Closing TCP listener...
2020-06-12 22:13:37.5859|DEBUG:Butler: While accepting connection: accept tcp 127.0.0.1:51522: use of closed network connection
2020-06-12 22:13:37.5859|DEBUG:Butler: Waiting for TCP connections to close...
2020-06-12 22:13:37.6362|DEBUG:Butler: All TCP connections closed
2020-06-12 22:13:40.8649|INFO:[db prepare] [debug] Current DB version is 1542741863
2020-06-12 22:13:40.8649|INFO:[db prepare] [debug] Latest migration is   1542741863
2020-06-12 22:13:40.8649|INFO:[db prepare] [debug] No migrations to run
2020-06-12 22:13:40.8649|DEBUG:{"secret":"249e88f8-629d-4c61-a34c-3a3646d6a98cf03d909a-e8c7-4d30-b7e3-e6248754283f30ab11f9-38e7-4727-8dfc-48c601b495ba266e90d9-997b-4226-aff9-e4172b786088","tcp":{"address":"127.0.0.1:51528"},"time":1592018020,"type":"butlerd/listen-notification"}
2020-06-12 22:13:43.5589|INFO:[router] [info] Initiating graceful butlerd shutdown
2020-06-12 22:13:43.5589|INFO:[router] [info] While shutting down, request 1557528757 has completed
2020-06-12 22:13:43.5589|INFO:[router] [info] No in-flight requests left, we can shut down now.
2020-06-12 22:13:43.5589|DEBUG:Butler: Closing TCP listener...
2020-06-12 22:13:43.5589|DEBUG:Butler: While accepting connection: accept tcp 127.0.0.1:51528: use of closed network connection
2020-06-12 22:13:43.5619|DEBUG:Butler: Waiting for TCP connections to close...
2020-06-12 22:13:43.6089|DEBUG:Butler: All TCP connections closed
2020-06-12 22:20:45.6893|INFO:Installing Captain MaCaw, d170043b-699e-4ff9-8d85-285bd5eecc70, 357687, 00000001-ebb2-4eec-abcb-7c89937a42bb
2020-06-12 22:20:45.9230|DEBUG:Opening URL: itch://library/owned
2020-06-12 22:20:47.7049|INFO:[db prepare] [debug] Current DB version is 1542741863
2020-06-12 22:20:47.7049|INFO:[db prepare] [debug] Latest migration is   1542741863
2020-06-12 22:20:47.7049|INFO:[db prepare] [debug] No migrations to run
2020-06-12 22:20:47.7049|DEBUG:{"secret":"55f02e80-5b22-4c7c-b082-91160b568b0b9075b354-ffeb-4bc6-8e94-a2902b89dfc99b6a4243-ef2d-4258-b517-71e63c2296bf3d65bc2c-430b-435e-89e1-9b5e81db38e8","tcp":{"address":"127.0.0.1:51573"},"time":1592018447,"type":"butlerd/listen-notification"}
2020-06-12 22:20:51.6450|DEBUG:Refreshing collection view filter.
2020-06-12 22:20:53.1960|DEBUG:Refreshing collection view filter.
2020-06-12 22:21:27.4672|DEBUG:Butler: jsonrpc2: protocol error: read tcp 127.0.0.1:51573->127.0.0.1:51574: wsarecv: Se produjo un error durante el intento de conexiÃ³n ya que la parte conectada no respondiÃ³ adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexiÃ³n establecida ya que el host conectado no ha podido responder.
```
Translation of last line: "An error occurred during the connection attempt because the connected party did not respond properly after a period of time, or an established connection failed because the connected host was unable to respond."

af80a7c9-017e-465a-a43d-f8e5abe45406

**Comment by JosefNemec** [comment]
I don't see any crash info in logs. Did it just close down or did it show crash dialog?

**Comment by darklinkpower** [comment]
Playnite freezed completely and I had to close it with task manager. Diag. package was made upon reboot.

**Comment by JosefNemec** [comment]
So it was not a crash then, how long did you wait before closing it down? Did it freeze before opening itch client or after?

**Comment by JosefNemec** [comment]
Is this reproducible on demand or was it a time thing?

**Comment by darklinkpower** [comment]
Not completely sure but I think it freezed very shortly before the itch.io client showed up. In fact, the installation launch was unintentional since it seems I clicked it twice in grid view. I thought the freeze would go away when the main itch.io window loaded completely but it didn't even after a minute.

It seems it was a one time thing because I can't find how to reproduce it. The crash might not even be related to itch.io's integration but to Playnite itself. I'll update here if I find something.

**Comment by JosefNemec** [comment]
Should be fixed in 8.4.
